homeassistant/components/zeroconf/__init__.py

```diff
@@ -224,11 +224,12 @@ def service_update(zeroconf, service_type, name, state_change):
                 )
             )
 
-    for service in ZEROCONF:
-        HaServiceBrowser(zeroconf, service, handlers=[service_update])
+    types = list(ZEROCONF.keys())
```

Comment:
Copying a dict to a list copies the keys by default. We can remove `.keys()`.

Reply:
Fixed.  Still need to get used to that.